test/support/factory.ex

```diff
@@ -475,6 +458,27 @@ defmodule Trento.Factory do
     }
   end
 
+  def hana_cluster_node_factory do
+    %HanaClusterNode{
+      attributes: %{"attribute" => Faker.Beer.name()},
+      hana_status: "Secondary",
+      virtual_ip: Faker.Internet.ip_v4_address(),
+      name: Faker.StarWars.character(),
+      indexserver_actual_role: nil,
```

Comment:
I would put some value instead of nil, as nil is a corner case. `master` and `master` is the most common scenario

Reply:
you're right! done